rainier-plot/src/main/scala/com/stripe/rainier/plot/Jupyter.scala

```diff
@@ -205,7 +206,7 @@ object Jupyter {
       Bounds(minX, maxX),
       Bounds(minY, maxY),
       new PlotRenderer {
-        def render(plot: Plot, plotExtent: Extent)(implicit theme: Theme) = {
+        def render(plot: Plot, plotExtent: Extent)(implicit _theme: Theme) = {
```

Comment:
This seems a little bit hacky to me - I feel like the intent for evilplot is to have a consistent theme throughout that is deferred until render time, and here you're ignoring the provided theme in favor of a "hardcoded" one. Seems better to me to have an explicit color parameter (that is optional and defaults to the theme's).

Reply:
Are you sure that's the intent of the EvilPlot API? Our `shade` feels analogous to something like `ScatterPlot.apply` or `BarChart.apply` both of which take an implicit theme. I believe the intent is that the theme passed to each plot configures that plot and things like the `pointRenderer` optional argument are only needed if you want to customize per-point.

Reply:
Ok, I buy that.